examples/compression/Cargo.toml

```
@@ -0,0 +1,22 @@
+[package]
+name = "compression"
```

All the examples are named `example-*` because they are part of the same workspace as the normal project. I don't think it should be named `compression` as that could confuse people into thinking this is something else than an example.

```suggestion
name = "example-compression"
```

---

https://github.com/tokio-rs/axum/pull/2623/commits/46a6276247e2f779cc16d98304657b0a40d8d322